Commons/src/main/java/com/github/jsonldjava/core/NGSIObject.java

```diff
@@ -705,10 +728,9 @@ private void validateAttribute(int payloadType, String expandedProperty, String
 			if (isVocabProperty && !hasVocab) {
 				throw new ResponseException(ErrorType.BadRequestData,
 						"You can't have vocabulary property without a vocab");
-			}
-			if (isListProperty && !hasList) {
+			}if (isJsonProperty && !hasJson) {
 				throw new ResponseException(ErrorType.BadRequestData,
-						"You can't have ListProperty property without a valueList");
```

Comment:
Validate exception in case of ListProperty

Reply:
done